.github/workflows/asana-release.yml

```diff
@@ -105,145 +117,44 @@ jobs:
                   path: autofill/
             - uses: ./autofill/.github/actions/setup-release-scripts
             # --- Action ---
-            - name: Checkout BSK
+            - name: Checkout Apple monorepo
               uses: actions/checkout@v3
               with:
-                  repository: duckduckgo/BrowserServicesKit
-                  path: bsk/
+                  repository: duckduckgo/apple-monorepo-test # TODO: Update with final monorepo URL
```

Comment:
The final repo now has history imported to it, so I'll update this on Thursday and run one more test.